### PR TITLE
libvirt: Fix PEP8 W503

### PR DESCRIPTION
**Pull request by Hao-Liu**
Signed-off-by: Hao Liu <hliu@redhat.com>

### CODE REVIEW

File: libvirt/tests/src/guest_kernel_debugging/nmi_test.py [hunk 1]
```diff
@@ -45,8 +45,8 @@ def run(test, params, env):
 
     if not libvirt_version.version_compare(1, 1, 1):
         if params.get('setup_libvirt_polkit') == 'yes':
-            raise error.TestNAError("API acl test not supported in current"
-                                    + " libvirt version.")
+            raise error.TestNAError("API acl test not supported in current" +
```
Comment (waynesun09) [inline libvirt/tests/src/guest_kernel_debugging/nmi_test.py]: The operator '+' is not necessary, implicit line joining inside parentheses could be all right.
@Hao-Liu what do you think?

**Comment by waynesun09** [comment]
Lot of the W503 problems found out here are caused by me, sigh...

**Comment by waynesun09** [comment]
ACK

**Comment by waynesun09** [comment]
@yumingfei @will-Do Can you guys help check this, as all PR will fail without W503 fix here.

**Comment by will-Do** [comment]
ACK and merge this fix, thanks

**Pull request merged**